content/doc/administration/requirements/jenkins-on-java-11.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: documentation
+title: Running Jenkins on Java 11
+---
+
+If you need to upgrade the JVM version used to run Jenkins from Java 8 to Java 11, please link:/doc/administration/requirements/upgrade-java-guidelines[follow this guidelines].
+
+NOTE: At this moment, it is not required to upgrade the JVM version in order to run a Jenkins core more recent than ``2.164+``footnote:[It was possible to run Jenkins on Java 11 since the version `2.155` but its usage was then more complex. We do not recommend to use any version prior to `2.164`.].
```

Comment:
This note is mixed AFAICT. First sentence tells that Jenkins still supports Java 8, this is good. Regarding the rest, I am not sure it is really needed. It might be better to just say that the guidelines apply to Jenkins 2.164+ in the paragraph above

Reply:
The footnote is really displayed at the end of the page, shouldn't be a problem.